code/ImageGalleryPage.php

```diff
@@ -157,6 +157,8 @@ public function getCMSFields() {
 				)
 			);
 		}
+
+		$this->extend('updateCMSFields', $fields);
```

Comment:
this will mean updateCMSFields is called twice, since it's called the first time in SiteTree::getCMSFields